### DISCUSSION TITLE
Notifications: More detailed selection of events

### DISCUSSION THREAD

**Discussion by addvanced** [coollabsio/coolify#2746] [Feature Requests]
Currently, we have the options to subscribe to events and get notifications when:
- Container Status Changes
- Application Deployments
- Backup Status
- Scheduled Tasks Status

But, for e.g. Backup Status and Scheduled Tasks Status, I would like to only get notifications when something goes wrong.
Maybe someone would like the same with application deployments.

It would be nice if it was possible to get a bit more specific options for each of these events, so I can decide if I want ALL notifications, or only "Successful" event or "Error" events etc.?

You would then have a checkbox next to the event "parent" (Backup Status), and then a checkbox for each of the child-events.
By checking the parent-event checkbox, it would then automatically check all child-events. And of course, if I manually checked all child-events, it would also check the parent-event checkbox - and uncheck everything if I don't want any event notifications.

But I could also just check a single child-event.

So, let's say for "Backup Status":

**Example 1: All selected**
- [x] Backup Status 
  - [x] Successful
  - [x] Error

**Example 2: Only Error events**
- [ ] Backup Status 
  - [ ] Successful
  - [x] Error

**Comment by stefanofusai** [comment]
Would love to see this implemented!
